code/src/App.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 
+import Homepage from 'Components/Homepage'
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Homepage />
```

Comment:
Everything in `<Homepage />` can be moved into `App.js` There's no reason to have it as a separate component. It just creates an additional file without good reason. It makes sense to have `App` as the core of the project with the route configuration etc.